Formula/libcapn.rb

```diff
@@ -20,58 +20,32 @@ class Libcapn < Formula
     sha256 "70d7e47ff2ad168c6f26e61d86805b4e0d1c37015c1e3528d589195a66e9d185" => :mavericks
   end
 
-  devel do
-    url "https://github.com/adobkin/libcapn/archive/2.0.0-beta.tar.gz"
-    sha256 "551dccfa66b616a390e3c9fc8ac35869a076c979246728e9343f9eeb50d66551"
-    version "2.0.0-beta"
-
-    resource "jansson" do
-      url "https://github.com/akheron/jansson.git", :revision => "e44b2231b50aea5de78b7ea2debec0d5327cd711"
-    end
-  end
-
   depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "openssl"
 
   def install
     if build.stable?
```

Comment:
Not a hard one, no. It makes sense to peg it to a specific commit for stable/devel & roam around freely for `head`, IMO. This _seems_ reasonable to me as-is.